docs/src/upgrade/release-3.7.x.asciidoc

```diff
@@ -146,6 +146,24 @@ See: link:https://issues.apache.org/jira/browse/TINKERPOP-2819[TINKERPOP-2819]
 distributions. Any app which makes use of both `gremlin-driver` and `gremlin-server` will now need to directly
 include both modules.
 
+==== Building and Running with JDK 17
```

Comment:
Should we also add a section in the developer doc section for this?

Reply:
Yeah, that is a good idea.

Reply:
i suspect there are more places in the docs that make various proclamations about JDK versions. please search around for a bit and see what might need updating.

Reply:
I added a small section to the developer documentation that mentioned running with various JDK versions.

> i suspect there are more places in the docs that make various proclamations about JDK versions. please search around for a bit and see what might need updating.

I looked around for this but most of the places that mention which Java runtime to use are generic and don't refer to a specific version. In the areas that specifically mention using Java 11 (like building documentation), I think it makes sense to keep them like that for now as I don't think JDK 17 will become our default choice yet.